src/query/runner.cr

```diff
@@ -0,0 +1,68 @@
+# Query runner which finalizes a query and runs it.
+# This will likely require adapter specific subclassing :[.
+class Query::Runner(T)
+  def initialize(@query : Compiled(T))
+  end
+
+  def log(*args)
```

Comment:
This will need to hook into a logger eventually, right? So it doesn't log in production.

Reply:
yep, it's just a stub for now